From 48514b41927f8601c6e1af7ebcf49c4af9a16b54 Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Fri, 19 Apr 2024 14:14:31 -0700
Subject: [PATCH 1/2] Exclude test directories from renaming in autopkgtest by
 default

* Files under test directories may also include utilities that are
used in tests but don't have any test in them.  It makes sense to keep
them by default during autopkgtest to make it work out-of-the-box.
---
 dh_elpa_test | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dh_elpa_test b/dh_elpa_test
index 14e31dd..b9f1152 100755
--- a/dh_elpa_test
+++ b/dh_elpa_test
@@ -271,7 +271,9 @@ if ($autopkgtest) {
     my $rule = File::Find::Rule->new;
     $rule
       ->or(File::Find::Rule
-           ->name('.pc', 'debian', '.git')
+           ->name('.pc', 'debian', '.git', # exclude non-source directories
+                  'test', 'tests', # exclude test directories
+           )
            ->directory->prune->discard,
            File::Find::Rule->new);
     $rule
-- 
2.39.2

